h5p-bildetema/src/components/CollectionsController/CollectionController.tsx

```diff
@@ -67,6 +70,12 @@ const CollectionController = ({
     );
   };
 
+  useEffect(() => {
+    // Scroll into view on load and if collection param changes (i.e. when
+    // switching between the main collection page and a collection)
+    h5pInstance?.getWrapper().scrollIntoView();
+  }, [collection, h5pInstance]);
```

Comment:
Also suggestion, for  getWrapper().scrollIntoView();  incase the DOM element or object does fail. Alternatively it can be done with a try catch, but I think that is unesseray. This helps to prevent errors that might occur if the DOM element is not attached properly,

```
useEffect(() => {
    const wrapper = h5pInstance?.getWrapper();
    if (wrapper) {
      wrapper.scrollIntoView();
    }
  }, [collection, h5pInstance]);
```

Reply:
Yes, I agree. Suggestion added